src/components/changeset/details.js

```diff
@@ -25,19 +28,40 @@ export function Details({
   );
 
   let sourceMatch = [];
-
   if (source && source.indexOf('{switch:a,b,c}.') > -1) {
     source = source.replace('{switch:a,b,c}.', '');
   }
   if (source && source.match(urlRegex)) {
     sourceMatch = source.match(urlRegex);
     source = source.replace(urlRegex, '');
   }
+
   let imageryMatch = [];
   if (imagery && imagery.match(urlRegex)) {
     imageryMatch = imagery.match(urlRegex);
     imagery = imagery.replace(urlRegex, '');
   }
+
+  const propertiesObj = {};
+
+  propertiesObj['editor'] = editor;
```

Comment:
It would be better to define `propertiesObj` with these 3 entries.
```
  const propertiesObj = {
     editor: editor,
     imagery: imagery,
     source: source
  };
```

Reply:
Done